rootfs/usr/local/bin/setup.sh

```diff
@@ -309,6 +309,26 @@ if [ -f /var/mail/postfix/custom.conf ]; then
   echo "[INFO] Custom Postfix configuration file loaded"
 fi
 
+#
+# Initialize rbl_override configuration
+#
+if [ -d "/etc/postifx/rbl_override" ]; then
+  echo "[ERROR] The `/etc/postfix/rbl_override` is a folder instead of a file.  Please fix it."
+  exit 1;
+fi
+
+if [ ! -f "/etc/postfix/rbl_override" ]; then
+  touch /etc/postfix/rbl_override
+  echo "[INFO] Created default (empty) rbl_override file"
+else
+  echo "[INFO] /etc/postfix/rbl_override file found"
+fi
+
+if [ -f "/etc/postfix/rbl_override" ]; then
```

Comment:
Under which conditions this if condition evaluates to false? I think we are pretty much making sure that it's true in the code above?

Reply:
Only if the `touch` command will fail above.

Reply:
Alright. I would personally have `set -e` at the top and fail on all unexpected errors. But this is a minor thing probably not worth the time spent talking about it ;)